dbwrapper/templates/dbwrapper/donation_form.html

```diff
@@ -170,6 +170,17 @@ <h3 class="panel-title">Pagamento</h3>
         <button class="submit-button" name="subbtn" onclick="ga('send', {hitType: 'event',eventCategory: 'doacao',eventAction: 'submit_donation',eventLabel: 'website'});" type="submit" >
             Doar
         </button>
+        <br/>
```

Comment:
Are we following any indentation pattern? I see some lines with 4 spaces, others with 2. I'd pick one and follow it, even if not there's no lint here.

Also, avoid using `br`s for styling. Use css's margin or padding for that